cypress/e2e/facility_spec/facility_manage.cy.ts

```diff
@@ -15,7 +15,7 @@ describe("Facility Manage Functions", () => {
     "Facility updated successfully";
   const facilityHrfidUpdateButton = "Link Health Facility";
   const facilityHrfidSuccessfullNotification =
-    "Health Facility config updated successfully";
+    /Health Facility config updated successfully|Health ID registration failed/;
   const facilityHrfId = uuidv4();
```

Comment:
Isn't it HFR ID?

Reply:
It can be any string value, it shouldn't be a problem, I will change it to a better-suited string.